docs/Catalog/VisualizationDashboards.md

```diff
@@ -285,7 +286,52 @@ Load More to fetch up to about 6MB of zipped data. Beyond this size, click Downl
 to see the entire file contents.
 
 In order to open the analysis and visualization capabilities of Perspecitve,
-click the vertical ellipsis, upper left. Click, drag, and drop columns to explore
-the data.
+click the vertical ellipsis, upper left. You can hold this settings control opened
+on page load in files showed with `quilt_summarize.json` by providing additional
+configuration:
+
+```json
+// quilt_summarize.json
+[
+  {
+    "path": "file1.csv",
+    "types": [
+      {
+        "name": "perspective",
+        "config": {
+          "settings": true
+        }
+      }
+    ]
+  }
+]
+```
+
+Besides `{ "settings": true }` you can provide any object previously saved from Perspective.
+
+![](../imgs/perspective-save.png)
+
+All filters and columns will be restored:
+
+```json
+// quilt_summarize.json
+[
+  {
+    "path": "file1.csv",
+    "types": [
+      {
+        "name": "perspective",
+        "config": {
+          "columns": ["name", "value"],
+          "group_by": ["value"],
+          "settings": true
+        }
+      }
+    ]
+  }
+]
+```
+
+Click, drag, and drop columns to explore the data.
```

Comment:
Drag columns from the left to the
```suggestion
Drag columns from the left to the top to sort, filter, and pivot.
Use the menu upper left to try different visualizations.
Use the controls along the bottom to download, copy, resize, and more.
```
Below is a better and crushed image for you (shows more), you can replace perspective-save with this.
<img width="800" alt="perspective-controls" src="https://user-images.githubusercontent.com/1322715/162036945-4d3ff963-5e63-475f-8280-5c46f38751d4.png">